pyaedt/emit.py

```diff
@@ -100,6 +332,9 @@ def __init__(
         port=0,
         aedt_process_id=None,
     ):
+        if projectname is None:
+            projectname = generate_unique_project_name()
+        self.__emit_api_enabled = False
         """Constructor."""
```

Comment:
```suggestion
        """Constructor."""
```
Most cases I have seen have something like this: "Constructor for the ``x`` class", where x is the class name.